src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,60 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+};
+
+export const MovieCard: React.FC<Props> = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => (
+  <article
+    className="card"
+    data-cy="Movie"
+  >
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          data-cy="MovieImage"
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8" data-cy="MovieTitle">
+            {title}
+          </p>
+        </div>
+      </div>
+
+      <div className="content">
+        <p data-cy="MovieDescription">
+          {description}
+        </p>
+
+        <a
+          href={imdbUrl}
+          data-cy="MovieLink"
+        >
+          IMDB
+        </a>
+      </div>
+    </div>
+  </article>
```

Comment:
`article `is not correct semantic tag for list rendering, you should use `ul` and `li` tags